src/vm/zapsig.cpp

```diff
@@ -1324,19 +1324,44 @@ BOOL ZapSig::EncodeMethod(
     {
         Module * pReferencingModule = pMethod->IsNDirect() ?
```

Comment:
Please document why `pReferencingModule` is valid to be `NULL`. Based on existing code this is a core assumption that is no longer value. If there is a way I would strong suggest to remove using `NULL` as an invariant since this function was written where that was never an issue.

Reply:
In particular see previous logic at line 1342.

Reply:
The problem is that there isn't a referencing module. Dynamic methods don't really have associated modules in the same way that normal methods do.  (In particular, the tokens are not relative to the module, which is what all of the code here is assuming.)

Reply:
But a comment is certainly worth it.